src/renderers/webxr/WebXRManager.js

```diff
@@ -191,6 +192,24 @@ class WebXRManager extends EventDispatcher {
 
 		};
 
+		this.getOffsetReferenceSpace = function () {
+
+			return offsetReferenceSpace;
+
+		};
+
+		this.setOffsetReferenceSpace = function ( space ) {
+
+			offsetReferenceSpace = space;
+
+		};
+
+		this.resetOffsetReferenceSpace = function () {
+
+			offsetReferenceSpace = null;
+
+		};
```

Comment:
I don't think we need this one.
We usually follow this pattern: `setOffsetReferenceSpace( null )`.

Reply:
Got it. Will remove this one.